docs/sources/guides/basic_concepts.md

```diff
@@ -4,117 +4,92 @@ description = "Grafana intro and concept guide"
 keywords = ["grafana", "intro", "guide", "concepts"]
 type = "docs"
 [menu.docs]
-name = "Basic Concepts"
+name = "Basic concepts"
 identifier = "basic_concepts"
 parent = "guides"
 weight = 2
 +++
 
-# Basic Concepts
+# Basic concepts
 
-This document is a “bottom up” introduction to basic concepts in Grafana, and can be used as a starting point to get familiar with core features.
+This document is an introduction to basic concepts in Grafana. Use it as a starting point to get familiar with core Grafana features.
 
-### Data Source
+## Dashboard
 
-Grafana supports many different storage backends for your time series data (Data Source). Each Data Source has a specific Query Editor that is customized for the features and capabilities that the particular Data Source exposes.
+The *dashboard* is where it all comes together. A dashboard is a set of one or more panels organized and arranged into one or more rows.
 
-The following data sources are officially supported:
+The time period for the dashboard can be controlled by the [Time range controls]({{< relref "timerange.md" >}}) in the upper right of the dashboard.
 
-* [CloudWatch]({{< relref "../features/datasources/cloudwatch.md" >}})
-* [Elasticsearch]({{< relref "../features/datasources/elasticsearch.md" >}})
-* [Graphite]({{< relref "../features/datasources/graphite.md" >}})
-* [InfluxDB]({{< relref "../features/datasources/influxdb.md" >}})
-* [OpenTSDB]({{< relref "../features/datasources/opentsdb.md" >}})
-* [Prometheus]({{< relref "../features/datasources/prometheus.md" >}})
+Dashboards can use [templating]({{< relref "templating.md" >}}) to make them more dynamic and interactive.
 
-The query language and capabilities of each Data Source are obviously very different. You can combine data from multiple Data Sources onto a single Dashboard, but each Panel is tied to a specific Data Source that belongs to a particular Organization.
+Dashboards can use [annotations]({{< relref "annotations.md" >}}) to display event data across panels. This can help correlate the time series data in the panel with other events.
 
-### Organization
+Dashboards can be [shared]({{< relref "share_dashboard.md" >}}) easily in a variety of ways.
 
-Grafana supports multiple organizations in order to support a wide variety of deployment models, including using a single Grafana instance to provide service to multiple potentially untrusted Organizations.
+Dashboards can be tagged, and the dashboard picker provides quick, searchable access to all dashboards in a particular organization.
 
-In many cases, Grafana will be deployed with a single Organization.
+## Data source
 
-Each Organization can have one or more Data Sources.
+Grafana supports many storage backends for your time series *data source*. Each data source has a specific Query Editor that is customized for the features and capabilities that the particular data source exposes.
```

Comment:
Grafana can visualize, explore & alert on data from many different databases and cloud services. Each database or service type are accessed via a **data source**

Reply:
Updated.